toolkit/tools/internal/userutils/userutils.go

```diff
@@ -76,14 +76,20 @@ func UserExists(username string, installChroot safechroot.ChrootInterface) (bool
 	return userExists, nil
 }
 
-func AddUser(username string, hashedPassword string, uid string, installChroot safechroot.ChrootInterface) error {
+func AddUser(username string, homeDir string, primaryGroup string, hashedPassword string, uid string, installChroot *safechroot.Chroot) error {
```

Comment:
Why did you change `installChroot` from `ChrootInterface` to `*safechroot.Chroot`? Please revert.

Reply:
Sorry, didn't see that when I was going through the changes. Will revert.